src/category_theory/triangulated/rotate.lean

```diff
@@ -0,0 +1,263 @@
+/-
+Copyright (c) 2021 Luke Kershaw. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Luke Kershaw
+-/
+import category_theory.additive.basic
+import category_theory.shift
+import category_theory.abelian.additive_functor
+import category_theory.natural_isomorphism
+import category_theory.triangulated.basic
+
+/-!
+# Rotate
+
+This file adds the ability to rotate triangles and triangle morphisms.
+It also shows that rotation gives an equivalence on the category of triangles.
+
+-/
+
+noncomputable theory
+
+open category_theory
+open category_theory.preadditive
+open category_theory.limits
+
+universes v v₀ v₁ v₂ u u₀ u₁ u₂
+
+namespace category_theory.triangulated
+open category_theory.category
+
+/--
```

Comment:
Using a `/--` comment here doesn't really make sense to me, since `/--` is intended for documenting definitions and theorems. If you want this comment to appear in the API documentation, then it should start with `/-!`, otherwise it should start with `/-` (it appears that using a `/--` in front of something that is not a declaration behaves the same way as `/-` right now). The same remark applies to the corresponding comment in the file `triangulated/basic.lean`.